dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -97,14 +115,24 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
     avg_save_time = (end - start) / num_save_calls
     num_load_calls = int(os.environ.get("NUM_LOAD_CALLS", 3))
     load_checkpoint_times = []
+    if args.save_only:
+        print("Skipping loads because you set --save_only")
+        num_load_calls = 0
+        load_checkpoint_times = [0]
+    if args.strategy == FSDP_STRATEGY and args.load_only:
+        print(f"Copying contents of {ckpt_dir_path} to {ckpt_restore_path}")
+        copy_bucket_to_local(ckpt_dir_path.removeprefix("gs://"),
+                             os.path.dirname(ckpt_restore_path))
+        avg_save_time = 0
```

Comment:
You should probably gate the printout for save/load with the same flag below so the output isn't confusing with the 0 seconds.

Reply:
How about I just print "skipped" instead of "0 seconds"

Reply:
Done.

Reply:
https://screenshot.googleplex.com/6FjQKCzc7wpjbNu